app/controllers/admin/todolist_controller.rb

```diff
@@ -1,6 +1,11 @@
 class Admin::TodolistController < Admin::AdminController
-
   def index
-    @petitions = Petition.for_state(Petition::SPONSORED_STATE).order(:created_at).paginate(:page => params[:page], :per_page => params[:per_page] || 20)
+    if params[:petition] == 'collecting_sponsors'
+      petitions = Petition.collecting_sponsors
+    else
+      petitions = Petition.in_moderation
+    end
+    @petitions = petitions.order(:created_at).paginate(page: params[:page], per_page: params[:per_page] || 20)
```

Comment:
Yes that's better! Thanks @pixeltrix, changed here: https://github.com/alphagov/e-petitions/commit/e83ad6e3557d5650db468135ab43a1d4e7fc515a